echo/frontend/src/components/announcement/utils/dateUtils.ts

```
@@ -0,0 +1,38 @@
+  "en-US": enUS,
+  "nl-NL": nl,
+  "de-DE": de,
+  "fr-FR": fr,
+  "es-ES": es,
+} as const;
+
+type SupportedLocale = keyof typeof localeMap;
+
+export const formatDate = (
+  date: string | Date | null | undefined,
+  locale: string = "en-US",
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)